google-cloud-error_reporting/.rubocop.yml

```diff
@@ -3,6 +3,7 @@ AllCops:
     - "integration/**/*"
     - "google-cloud-error_reporting.gemspec"
     - "lib/google/devtools/**/*"
+    - "lib/google/cloud/error_reporting/**/*"
```

Comment:
This will exclude the hand-coded files [lib/google/cloud/error_reporting/middleware.rb](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-error_reporting/lib/google/cloud/error_reporting/middleware.rb) and [lib/google/cloud/error_reporting/rails.rb](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-error_reporting/lib/google/cloud/error_reporting/rails.rb) from rubocop checks.